### ISSUE TITLE
Suggestions for model training

### ISSUE THREAD

**Issue by mkutu** [precision-sustainable-ag/Field-SegmentationTraining#19]
After other issues have been closed:

- [ ] drop the last validation batch to avoid issues with batch normalization or loss averaging when the last batch is smaller than your set batch size
```python 
   self.val_loader = DataLoader(self.val_dataset, batch_size=self.batch_size, shuffle=False, drop_last=True)
```
- [ ] play with influence of cropping/resizing on IoU and visual inspection results
